iceoryx_posh/source/roudi/roudi.cpp

```diff
@@ -235,11 +235,30 @@ void RouDi::processMessage(const runtime::IpcMessage& message,
             cxx::Serialization portConfigInfoSerialization(message.getElementAtIndex(7));
 
             popo::PublisherOptions options;
-            options.historyCapacity = std::stoull(message.getElementAtIndex(3));
+            uint64_t historyCapacity{};
+            if (!cxx::convert::fromString(message.getElementAtIndex(3).c_str(), historyCapacity))
+            {
+                LogError() << message.getElementAtIndex(3).c_str() << "cannot be extracted from string\n";
```

Comment:
Please add something like `Invalid parameter for \"IpcMessageType::CREATE_PUBLISHER\"! '` << ... << "' cannot ..." to know from which request the error originates and also the `'` to separate the received string from the error message.
Same with the other error logs.